spec/controllers/avail_links_controller_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe Preferences::AvailLinksController, type: :controller do
+  setup :activate_authlogic
+  before do
+    @account = Account.create! username: 'testaccount', password: '123456', password_confirmation: '123456'
+    @plan = Plan.create! account: @account, edit_text: ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.